.mocharc.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+
+const nodeOptions = ["experimental-vm-modules"];
+const [major, _minor, _patch] = process.versions.node.split('.');
+if (Number(major) >= 23) {
+  nodeOptions.push('no-experimental-strip-types');
+}
+
+module.exports =
+{
+  $schema: "https://raw.githubusercontent.com/SchemaStore/schemastore/master/src/schemas/json/mocharc.json",
```

Comment:
This was here only to get hinting in VSCode in the JSON

adding
```
/** @type {import("mocha").MochaOptions} */
```
on the line above module.exports will do the same

Reply:
@W-A-James missed this I think, or would you prefer to omit it?

Reply:
Fixed!